test/generic/LaurentPoly-test.jl

```diff
@@ -204,20 +218,27 @@ using AbstractAlgebra.Generic: Integers, LaurentPolyWrapRing, LaurentPolyWrap,
       @test !isdivisible_by(3*y+4, 2+3*y^-1)
    end
 
+   @testset "coercion" begin
+      R, x = PolynomialRing(ZZ, "x")
+      L, x1 = LaurentPolynomialRing(ZZ, "x")
+      @test L(x) == x1
+      @test L(x+x^2) == x1+x1^2
+   end
+
    @testset "comparisons" begin
       L, y = LaurentPolynomialRing(ZZ, "y")
       x = y.poly
 
       @test y == y
 
-      f = LaurentPolyWrap(x^3 + 2x^2 - 1)
+      f = LaurentPolyWrap(L, x^3 + 2x^2 - 1)
```

Comment:
Just for clarification. This is the same as `L(x^3 + 2x^2 - 1)`?

Reply:
Yeah. We should probably change that. I dont like so many internal fxns in the test code.

Reply:
Changed it.